frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/__tests__/WhenExpressionDecorator.spec.tsx

```diff
@@ -0,0 +1,102 @@
+import * as React from 'react';
+import { Tooltip } from '@patternfly/react-core';
+import { shallow, ShallowWrapper } from 'enzyme';
+
+import WhenExpressionDecorator from '../WhenExpressionDecorator';
+import { runStatus } from '../../../../utils/pipeline-augment';
+
+jest.mock('react-i18next', () => {
+  const reactI18next = require.requireActual('react-i18next');
+  return {
+    ...reactI18next,
+    useTranslation: () => ({ t: (key) => key }),
+  };
+});
+
+type WhenExpressionDecoratorProps = React.ComponentProps<typeof WhenExpressionDecorator>;
+
+describe('WhenExpressionDecorator', () => {
+  let wrapper: ShallowWrapper<WhenExpressionDecoratorProps>;
```

Comment:
nit: `whenExpressionDecoratorWrapper`?

Reply:
TBH,  `whenExpressionDecoratorWrapper` name is too long and wrapper is used to in many spots and i think we dont need to be specific about the wrapper as we are having only one component in this test.